src/main/g8/src/main/scala/Main.scala

```diff
@@ -1,4 +1,4 @@
-def main(args: Array[String]): Unit = {
+@main def Hello: Unit = {
```

Comment:
I think `def hello` would be more idiomatic?

Reply:
yes I think so, changed